test/unit/pixel/state.spec.ts

```diff
@@ -41,7 +42,7 @@ describe('EventComposition', () => {
       eventSource: { eventName: 'viewContent' },
       liveConnectId: '213245',
       trackerName: 'test tracker',
-      pageUrl: 'https://wwww.example.com?sss',
+      pageUrl: 'https://wwww.example.com/?sss',
```

Comment:
Why the escape here?

Reply:
The URL class I was using always set the pathname to `/` when empty, but I might be able to revert it now. I will check